tests/functional/binary-cache.sh

```diff
@@ -107,131 +107,135 @@ fi
 # Test whether fallback works if a NAR has disappeared. This does not require --fallback.
 clearStore
 
-mv $cacheDir/nar $cacheDir/nar2
+mv "$cacheDir"/nar "$cacheDir"/nar2
 
-nix-build --substituters "file://$cacheDir" --no-require-sigs dependencies.nix -o $TEST_ROOT/result
+nix-build --substituters "file://$cacheDir" --no-require-sigs dependencies.nix -o "$TEST_ROOT"/result
 
-mv $cacheDir/nar2 $cacheDir/nar
+mv "$cacheDir"/nar2 "$cacheDir"/nar
 
 
 # Test whether fallback works if a NAR is corrupted. This does require --fallback.
 clearStore
 
-mv $cacheDir/nar $cacheDir/nar2
-mkdir $cacheDir/nar
-for i in $(cd $cacheDir/nar2 && echo *); do touch $cacheDir/nar/$i; done
+mv "$cacheDir"/nar "$cacheDir"/nar2
+mkdir "$cacheDir"/nar
+for i in $(cd "$cacheDir"/nar2 && echo *); do touch "$cacheDir"/nar/"$i"; done
```

Comment:
I think shellcheck does not detect this, but we should probably use find here, too